grafana/dashboards/efficiency.json

```diff
@@ -181,8 +181,8 @@
         "#d44a3a"
       ],
       "datasource": "TeslaMate",
-      "decimals": 1,
-      "format": "kwatth",
+      "decimals": 0,
+      "format": "watth",
```

Comment:
The `/ 100km` (`/ 100mi`) postfix value also needs to be changed for both consumption panels :)

Reply:
Fixed these panels, thanks!